.github/workflows/ruby.yml

```diff
@@ -11,18 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [2.6, 2.7, 3.0, 3.1] # 2.6 is EOL March 31, 2022; kept for information only
-        rails: [5.2.6, 6.1.4.4, 7.0.1] # 5.2 EOL in 01 Jun 2022; kept for information only
-        exclude:
-        # excludes Rails 5.2.x on Ruby 3.0 and on 3.1 
-        # Note that 6.0.x should also exclude 3.0 and 3.1 but we're not testing it.
-        - rails: 5.2.6
-          ruby: 3.0
-        - rails: 5.2.6
-          ruby: 3.1
-        # Rails 7.0.1 requires >= 2.7.0, so excluding 2.6
-        - rails: 7.0.1
-          ruby: 2.6
+        ruby: [3.0, 3.1, 3.2] # 3.0 EOL is 2024-03-31
```

Comment:
Use "3.0" so it doesn't get converted to 3. Also, add 3.3.

Reply:
At this point 3.0 was EOL way too recently, so it's probably a good idea to keep support for it.